### ISSUE TITLE
ExpandRowColumn replacing grid when expanded

### ISSUE THREAD

**Issue by keithslater** [kartik-v/yii2-grid#488]
Hello, I am using ExpandRowColumn and for some reason when a row is expanded, the grid is totally replaced with the expanded data. I am using detailUrl and in the controller I am doing renderAjax. The only thing that it on the rendered page is a Highchart. I can go to the URL and it looks fine. No other HTML or anything besides the Highcharts and the associated js files.

Anyone know what would cause this to replace my grid?

**Comment by kartik-v** [comment]
You need to check your markup/code and other javascript plugins/code that should be causing this. Unfortunately cannot reproduce your use case.

**Comment by romany4** [comment]
I think you used something like this - 
`<?= Select2::widget([...]); ?>`
 in your code instead of 
`<?=
    $form->field($searchModel, 'param', [...])
        ->label(...)
        ->widget(Select2::classname(), [...]);
    ?>`
